types/node/index.d.ts

```diff
@@ -2180,7 +2180,7 @@ declare module "url" {
         host?: string;
         pathname?: string;
         search?: string;
-        query?: string | any;
+        query?: string | null | { [key: string]: string };
```

Comment:
You may get also `{ [key: string]: string | string[] }` for e.a. `?foo=bar&a=b&a=c`.

Reply:
Thanks, updated!